collectors/python.d.plugin/energid/energid.chart.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+# Description: Energi Core / Bitcoin netdata python.d module
+# Author: Andrey Galkin <andrey@futoin.org> (andvgal)
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+# This module is designed for energid, but it should work with many other Bitcoin forks
+# which support more or less standard JSON-RPC.
+#
+
+import json
+
+from bases.FrameworkServices.UrlService import UrlService
+
+update_every = 5
+
+ORDER = [
+    'blockindex',
+    'difficulty',
+    'mempool',
+    'secmem',
+    'network',
+    'utxo',
+]
+
+CHARTS = {
+    'blockindex': {
+        'options': [None, 'Blockchain Index', 'count', 'blockchain', 'energi.blockindex', 'area'],
+        'lines': [
+            ['blockchain.blocks', 'blocks', 'absolute'],
+            ['blockchain.headers', 'headers', 'absolute'],
+        ]
+    },
+    'difficulty': {
+        'options': [None, 'Blockchain Difficulty', 'difficulty', 'blockchain', 'energi.difficulty', 'line'],
+        'lines': [
+            ['blockchain.difficulty', 'Diff', 'absolute'],
+        ],
+    },
+    'mempool': {
+        'options': [None, 'MemPool', 'MiB', 'mempool', 'energi.mempool', 'area'],
+        'lines': [
+            ['mempool.max', 'Max', 'absolute', None, 1024*1024],
+            ['mempool.current', 'Usage', 'absolute', None, 1024*1024],
+            ['mempool.txsize', 'TX Size', 'absolute', None, 1024*1024],
+        ],
+    },
+    'secmem': {
+        'options': [None, 'Secure Memory', 'KiB', 'memory', 'energi.secmem', 'area'],
+        'lines': [
+            ['secmem.total', 'Total', 'absolute', None, 1024],
+            ['secmem.locked', 'Locked', 'absolute', None, 1024],
+            ['secmem.used', 'Used', 'absolute', None, 1024],
+        ],
+    },
+    'network': {
+        'options': [None, 'Network', 'count', 'network', 'energi.network', 'line'],
+        'lines': [
+            ['network.connections', 'Connections', 'absolute'],
+        ],
+    },
+    'utxo': {
+        'options': [None, 'UTXO', 'count', 'UTXO', 'energi.utxo', 'line'],
+        'lines': [
+            ['utxo.count', 'UTXO', 'absolute'],
+            ['utxo.xfers', 'Xfers', 'absolute'],
```

Comment:
Please use `_` instead of `.`. I recall this is needed for prometheus (exporting).

Reply:
There is still dot left in the chart names. It's OK, right?

Reply:
i don't see any dots :thinking: